src/precice/config/ParticipantConfiguration.cpp

```diff
@@ -495,12 +495,14 @@ void ParticipantConfiguration::finishParticipantConfiguration(
         for (mesh::PtrData data : meshContext.mesh->data()) {
           if (data->getName() == dataContext.fromData->getName()) {
             dataContext.toData = data;
+            dataContext.fromData = data;
```

Comment:
This should not be needed if we go for the "no-defined-mapping solution".

Reply:
Agree.